dynamic-demo-plugin/src/components/ProxyService.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { consoleFetchJSON } from '@openshift-console/dynamic-plugin-sdk';
+
+const ProxyService: React.FC = () => {
```

Comment:
nit: The two components names are too similar. I'd call this something like `ExampleProxyPage` and `ExampleProxyResponse`.